### DISCUSSION TITLE
【广告】凯木都市轻型车

### DISCUSSION THREAD

**Discussion by xiri61** [share121/inter-knot#554] [其他]
![微信图片_2024081011154 拷贝](https://github.com/user-attachments/assets/781d42ae-7d7d-47b6-9ad4-a3e3f6552b3f)
【广告】凯木都市轻型车，平静不平凡！你是否为车辆价格过高而苦恼？你是否为出行停车困难而心烦？ CALM—凯木都市轻型车，代步新传说，平静不平凡！ 现在购买， 尊享24期免息分期付款！（详情请咨询奥佛汽车线下门店）

**Comment by Tragedysaver1024** [comment]
我的评价是骑这辆三蹦子不如骑自行车

**Comment by FrostPhoenix79** [comment]
你就说多少丁尼，让我死心吧

**Comment by GUILAISHIXV** [comment]
支持信用点支付吗

**Comment by StarrySkyehk** [comment]
可以摩拉支付吗?

**Comment by Micawber-jock** [comment]
可以用晖长石号换吗

**Comment by wuzhijiaome** [comment]
市区限行吗？
